src/Cosmos/JsonSerialization.cpp

```diff
@@ -28,7 +28,7 @@ const string TYPE_PREFIX_MSG_WITHDRAW_REWARD = "cosmos-sdk/MsgWithdrawDelegation
 const string TYPE_PREFIX_PUBLIC_KEY = "tendermint/PubKeySecp256k1";
 const string TYPE_PREFIX_WASM_MSG_EXECUTE = "wasm/MsgExecuteContract";
 
-static string broadcastMode(Proto::BroadcastMode mode) {
+static string jsonBroadcastMode(Proto::BroadcastMode mode) {
```

Comment:
hmm, why add `json` prefix, it's under `sonSerialization.cpp`

Reply:
Because unity build include multiple c++ files to do a single compilation unit in order to speed up the build time, since those 2 c++ files have the same function name and get included in the same translation unit they conflict in name, either we rename the function either we make them belong to different namespace